packages/material-ui-lab/src/YearPicker/YearPicker.test.tsx

```diff
@@ -40,9 +27,17 @@ describe('<YearPicker />', () => {
       classes,
       inheritComponent: 'div',
       mount: localizedMount,
+      render,
+      muiName: 'MuiYearPicker',
       refInstanceof: window.HTMLDivElement,
       // cannot test reactTestRenderer because of required context
-      skip: ['componentProp', 'propsSpread', 'reactTestRenderer'],
+      skip: [
+        'componentProp',
+        'componentsProp',
+        'propsSpread',
+        'reactTestRenderer',
+        'themeDefaultProps',
```

Comment:
I need to skip themeDetaultProps because the current implementation does not receive "other" props. I tried to add "other" props to `YearPickerRoot` like in other component but then receive picker-related props that leak to DOM (ex, mask, inputFormat, ...)

Reply:
Looks like `propsSpread` is also skipped so it makes sense. Maybe we could introduce `testDefaultProps` option where we can specify some prop that exists in the API with specific value and test for it, for example `minDate` in this case.